### ISSUE TITLE
Protocol mismatch; server version = 11, client version = 10 on rails main when running the tests

### ISSUE THREAD

**Issue by dorianmariecom** [basecamp/solid_queue#169]
When I point to rails main, do `bundle update` then `rake` I get:

```
/Users/dorianmariefr/.asdf/installs/ruby/3.2.2/lib/ruby/gems/3.2.0/bundler/gems/rails-608c1bfe17a2/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb:35:in `rescue in new_client': Protocol mismatch; server version = 11, client version = 10 (ActiveRecord::ConnectionNotEstablished)
	from /Users/dorianmariefr/.asdf/installs/ruby/3.2.2/lib/ruby/gems/3.2.0/bundler/gems/rails-608c1bfe17a2/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new_client'
	from /Users/dorianmariefr/.asdf/installs/ruby/3.2.2/lib/ruby/gems/3.2.0/bundler/gems/rails-608c1bfe17a2/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb:145:in `connect'
	from /Users/dorianmariefr/.asdf/installs/ruby/3.2.2/lib/ruby/gems/3.2.0/bundler/gems/rails-608c1bfe17a2/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb:154:in `block in reconnect'
	from /Users/dorianmariefr/.asdf/installs/ruby/3.2.2/lib/ruby/gems/3.2.0/bundler/gems/rails-608c1bfe17a2/activesupport/lib/active_support/concurrency/null_lock.rb:9:in `synchronize'
```

**Comment by rosa** [comment]
Does this happen when you run MySQL in Docker via `bin/setup`?

**Comment by jeremy** [comment]
Mismatched MySQL versions. Using bin/setup resolved in #170.

**Comment by dorianmariecom** [comment]
Yeah didn't see there was a bin/setup
